setup.cfg

```diff
@@ -31,9 +31,10 @@ project_urls =
 [options]
 packages = find:
 install_requires =
-    numpy>=1.18.5; platform_machine!='aarch64' and platform_machine!='arm64' and python_version<'3.10'
+    numpy>=1.18.5; platform_machine!='aarch64' and platform_machine!='arm64' and platform_machine!='loongarch64' and python_version<'3.10'
     numpy>=1.19.2; platform_machine=='aarch64' and python_version<'3.10'
     numpy>=1.20.0; platform_machine=='arm64' and python_version<'3.10'
+    numpy>=1.22.0; platform_machine=='loongarch64'
     numpy>=1.21.0; python_version>='3.10'
```

Comment:
This doesn't look correct; the condition on this line should read:
```
numpy>=1.21.0; python_version>='3.10' and platform_machine!='loongarch64'
```
I believe. Otherwise line 37 and 38 are both matches for Python >= 3.10 on loongarch64.

Note that these conditions are quite fragile, `oldest-supported-numpy` has a test that ensures this kind of conflicting requirement won't occur. You may want to copy it.